libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -531,6 +547,20 @@ open class SalesforceSDKManager protected constructor(
      */
     var shouldBlockSalesforceIntegrationUser = false
 
+    /**
+     * Creates a NativeLoginManager instance that allows the app to use it's
+     * own native UI for authentication.
+     *
+     * @param consumerKey The Connected App consumer key.
+     * @param callbackUrl The Connected App redirect URI.
+     * @param communityUrl The login url for native login.
+     * @return The Native Login Manager.
+     */
+    fun useNativeLogin(consumerKey: String, callbackUrl: String, communityUrl: String): NativeLoginManagerInterface {
```

Comment:
As I mentioned above since we aliased "com.salesforce.androidsdk.auth.**_nativeLoginInterface_**.NativeLoginManager" to `NativeLoginManagerInterface` we can simply use that instead of having to write the _entire path_ for it OR the implementation class of the same name.  And the best part is the consumer still sees the return type as the interface `NativeLoginManager`.